torch/csrc/utils/tensor_qschemes.cpp

```diff
@@ -0,0 +1,39 @@
+#include <torch/csrc/utils/tensor_qschemes.h>
+
+#include <torch/csrc/DynamicTypes.h>
+#include <torch/csrc/Exceptions.h>
+#include <torch/csrc/QScheme.h>
+#include <c10/core/QScheme.h>
+
+#include <torch/csrc/python_headers.h>
+#include <torch/csrc/utils/object_ptr.h>
+
+
+namespace torch {
+namespace utils {
+
+#define _ADD_QSCHEME(qscheme, name)                                     \
+  {                                                                     \
+    std::string module_name = "torch.";                                 \
+    PyObject* qscheme_obj = THPQScheme_New(qscheme, module_name + name); \
+    Py_INCREF(qscheme_obj);                                             \
+    if (PyModule_AddObject(torch_module, name,qscheme_obj) != 0) {      \
```

Comment:
so you don't want this in a submodule?  That seems okay, but maybe these should start with "quant_" then?

Reply:
you mean `torch.quant_qscheme`? we can put this in `torch.quantized` or `torch.quantization` as well